release/RELEASE.md

```diff
@@ -3,12 +3,17 @@
 1. Make a PR that bumps the version number in the `VERSION`
    file and adds a new header and label for the new version in
    `docs/source/support/release-notes.rst` (see previous releases as examples).
-   Release notes should be cut and pasted under the new header from `unreleased.rst`.
-   Each change outlined in `unreleased.rst` is preceded by the section to
+   Release notes additions can be retrieved from commits using the following command:
+
+     for sha in $(git log --format=%H <revision range>); do git show --format=%b $sha | awk '/^$/{next} /^CHANGELOG$/{flag=1; next} flag'; done
```

Comment:
Can we add that in a bash script?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3547/commits/0fe067ccfc4601fdd08e43d32391a35e3e643f63

Reply:
This also gives a tool to check the sanity of the changelog additions of a PR by running `./unreleased.sh master..`